content/2. Ondersteunende-informatie/4. Techniek/Git/3. Branches.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 3. Branches
+draft: false
+---
+
+> Meer informatie over branches: [Git branches](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)
+## Wat zijn branches
+Een branch is een afzonderlijke lijn van ontwikkeling in een Git-repository. Bij het aanmaken van een repository komt altijd de branch `main` of `master` als hoofdbranch. Naast deze hoofdbranches kan je zelf ook branches erbij maken, zodat je kan werken aan de code zonder de hoofdcode te beïnvloeden.
```

Comment:
Misschien mooier en duidelijker om het iets als dit te maken:
"Een branch is een vertakking van de Git repository" + [voeg hier weer link toe naar git repository pagina] + "In deze vertakkingen kan je werken aan de code zonder de hoofdcode te beïnvloeden. Bij het aanmaken......"
Tweede ding, hoofdcode? Wat is dat dan?
Ja ik weet het wel, maar een eerstejaars of iemand zonder ervaring heeft geen idee wat hoofdcode vs code is